casper/src/main/resources/PoS.rhox

```diff
@@ -39,29 +40,38 @@ new  PoS,
 
   for(@(_, RevVault) <- revVaultCh;
       @(_, ListOps) <- listOpsCh) {
-    new posRevAddressCh in {
+    new posRevAddressCh, posAuthKeyCh, posVaultCh, initialActiveCh in {
 
+      @RevVault!("deployerAuthKey", *posAuthKeyCh) |
```

Comment:
I forgot why this works even though the current user is the validator that is executing this logic.

Reply:
The code is only called when the `PoS` contract is first deployed. That deploy uses the `PoS` contract's secret key.